src/test/resources/outputs/partiql/basics/aggregation.sql

```diff
@@ -1,5 +1,7 @@
 --#[rel-aggregation-count-00]
-SELECT COUNT(*) AS "_1" FROM "default"."T" AS "T";
+SELECT count(1) AS "_1" FROM "default"."T" AS "T";
+-- previously was using COUNT(*)
+-- SELECT COUNT(*) AS "_1" FROM "default"."T" AS "T";
```

Comment:
self-review: another change in behavior is that `COUNT(*)` maps to `COUNT(1)`. issue to add back `COUNT(*)` -> `COUNT(*)` support in the future -- https://github.com/partiql/partiql-scribe/issues/83